services/src/main/java/org/exoplatform/news/filter/NewsFilter.java

```diff
@@ -19,6 +19,8 @@ public class NewsFilter {
 
   private String       searchText;
 
+  private String       originText;
```

Comment:
Could you add the fuzzy index here instead of duplicating the original searched text ? this fuzzy index could b e assigned to the filter inside the class https://github.com/exoplatform/news/blob/e3b16f21e2377b1b58a165211e5a720afdd3d8a1/services/src/main/java/org/exoplatform/news/connector/NewsSearchConnector.java#L69